### ISSUE TITLE
Pip install errors

### ISSUE THREAD

**Issue by jw3** [ctc-oss/xview-lfs#2]
Client side error when consuming as dependency

```
error in xview_lfs setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'{-e git+'"
```

**Comment by jw3** [comment]
### problem

Editable pip entries like the following from freeze result in the noted error when loaded into `setup.py`

`-e git+https://github.com/ctc-oss/xview-utils@94a23e94fd3e03fa80ceec009e789d8613ed51df#egg=xview`

### solution
Use PEP 508 urls

`xview @ git+https://github.com/ctc-oss/xview-utils@94a23e94fd3e03fa80ceec009e789d8613ed51df`

Rewrite the entries when freezing for lack of a better solution

`pip freeze | sed -E 's/^-e (.+)\#egg=(.+)$/\2 @ \1/g' > requirements.txt`

### reference
https://www.python.org/dev/peps/pep-0508/
https://stackoverflow.com/a/54793503

**Comment by jw3** [comment]
fixed in 42f1929050c7bc8f86c78aea9c340895bbc4d217
